simple_history/models.py

```diff
@@ -222,6 +223,22 @@ def get_instance(self):
                 for field in fields.values()
             })
 
+        def get_next_record(self):
+            """
+            Get the next history record for the instance. `None` if last.
+            """
+            return self.instance.history.filter(
```

Comment:
Good feedback.  This forced me to implement these in a much more efficient and clean way.  Thanks!